### DISCUSSION TITLE
PostgREST Config Error

### DISCUSSION THREAD

**Discussion by HarshGaurup** [supabase/supabase#26403] [Questions] [answered]
I am stuck with something. I want to change the max rows value for our Supabase query from 1000 to more. The problem is that when I try to change the max rows, it shows the error "Failed to update postgrest config: An error has occured. Failed to fetch." Can you guys kindly help me with this if possible?

![image](https://github.com/supabase/supabase/assets/168011199/bfce2776-21c0-4c55-b6c6-35d465cbca47)

**Comment by GaryAustin1** [comment, marked as answer]
Is this hosted?
You might check your instance health on "home" page in the dashboard.
This is also caught people especially in Chrome with a certain extension... https://github.com/orgs/supabase/discussions/17296
